tests/net_util.proptest-regressions

```diff
@@ -1,7 +0,0 @@
-# Seeds for failure cases proptest has generated in the past. It is
```

Comment:
Why did we kill the regressions file?

Reply:
By mistake. :grimacing: I'll restore it.
I'm not sure what this one is for, though: Was there a bug in `network_dimensions_shrink_and_grow`?

Reply:
Frankly, I don't know how to read these =). I could look it up, but for now, it probably won't hurt to leave it in?